src/utils/replaceAccentChars.js

```diff
@@ -0,0 +1,10 @@
+export default function replaceAccentChars(str) {
```

Comment:
Why is this needed?

Reply:
This will replace accent characters like _ã_ from the search term and also from the `title` and `authorName` then it will be easy for the user search the right article.